config/daemonconfig.go

```diff
@@ -97,6 +97,7 @@ type BackendConfig struct {
 		Fallback      bool   `json:"fallback"`
 		PingURL       string `json:"ping_url,omitempty"`
 		CheckInterval int    `json:"check_interval,omitempty"`
+		UseHTTP       bool   `json:"use_http,omitempty"`
```

Comment:
The configuration in [here](https://github.com/containerd/nydus-snapshotter/blob/main/docs/configure_nydus.md) is obsolete, can we update it?